sdk/resourcemanager/Azure.ResourceManager/tests/Scenario/ArmClientTests.cs

```diff
@@ -203,7 +203,7 @@ public async Task GetUsedResourceApiVersionWithOverride()
             var subscription = await client.GetDefaultSubscriptionAsync();
             var version = await subscription.GetResourceProviders().GetApiVersionAsync(computeResourceType);
             Assert.AreEqual(expectedVersion, version);
-            Assert.AreEqual(0, policy.GetCount("Microsoft.Compute"));
+            Assert.AreEqual(1, policy.GetCount("Microsoft.Compute"));
```

Comment:
After the change, there will be one more API request to fetch API versions for `Microsoft.Compute`

Reply:
Can this be avoided (in some cases)?

Reply:
Yes, it can if the required resource type is in the override already.